CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Yii Queue Change Log
+# Yii Queue Extension Change Log
```

Comment:
Better call package as "Yii Queue" without pistfix "extension".

Reply:
"pistfix"... russian?

Reply:
Postfix. I mean rename "Yii Queue Extension" to "Yii Queue".

Reply:
It's not resolved.

Reply:
resolved in https://github.com/yiisoft/queue/pull/199/commits/a6392e7f84c379aeb6e91907d8bb56dcc7f2d2b3

Reply:
ok. done.